app/controllers/app_menu_list.js

```diff
@@ -1,44 +1,55 @@
-import Ember from 'ember';
-import config from '../config/environment';
+import Ember from "ember";
+import config from "stock/config/environment";
 const { getOwner } = Ember;
-import AjaxPromise from 'stock/utils/ajax-promise';
+import AjaxPromise from "stock/utils/ajax-promise";
+import createInventroyAction from "stock/mixins/create_Inventory";
+import storageType from "stock/mixins/storage-type";
 
-export default Ember.Controller.extend({
+export default Ember.Controller.extend(createInventroyAction, storageType, {
```

Comment:
No need of importing these mixins if you are directly calling its action from handlebars.